allink_core/core_apps/allink_content/models.py

```diff
@@ -143,12 +143,6 @@ class AllinkContentPlugin(CMSPlugin):
         blank=True,
         null=True
     )
-    project_on_screen_effect = models.CharField(
```

Comment:
when we remove this, we won't be able to add a "project_on_screen_effect" on project basis. Is this something we never want to do again? I would suggest that we keep the field on the content Plugin or the moment.

Reply:
The effect could still be added on project basis using the `CONTENT_CSS_CLASSES` in `settings.py`. If we only remove the frontend part then there is an option in the CMS that does nothing.

Reply:
yes true. but CONTENT_CSS_CLASSES is in my opinion less intuitiv or the CMS user. But ok, let's remove it. Less stuff to maintain, I like it :)